### PR TITLE
SCE-1140: cat experiment pivot ui improvements & latency table

### PR DESCRIPTION
**Pull request by ppalucki**
Fixes issue "there is no way to display memcached-cat experiment results"

Summary of changes:
- pivot_table ui improvementes (options + some sane default added)
- `latency()` table with that can be called like this `latency(aggressor='Caffe', qps=500000)`

Testing done:
- yes in production

examples:

simple call:

```python
cat.latency()
```

image

or you can filter some dimmensions:

```python
cat.latency(aggressor='stress-ng-stream', qps=375000)
```

image

### CODE REVIEW

**Comment by ppalucki** [comment]
run integration tests

**Comment by ppalucki** [comment]
run integration tests

**Comment by ppalucki** [comment]
run integration tests

File: jupyter/swan.py [hunk 1]
```diff
@@ -704,31 +735,53 @@ def __init__(self, experiment, slo):
...
+        columns = [
              # 'swan_repetition',
-             'swan_loadpoint_qps',
-             'swan_aggressor_name',
-             'be_number_of_cores',
-             'be_l3_cache_ways',
+             SWAN_LOAD_POINT_QPS_LABEL,
+             SWAN_AGGRESSOR_NAME_LABEL,
+             BE_NUMBER_OF_CORES_LABEL,
+             BE_L3_CACHE_WAYS_LABEL,
```
Comment (iwankgb) [inline jupyter/swan.py]: @ppalucki - could you remove commented out lines below, please?

File: jupyter/swan.py [hunk 2]
```diff
@@ -742,20 +795,17 @@ def simple_df(self):
...
-             'llc_occupancy/be/bytes',
-             'llc_occupancy/be/perecentage',
+        ]
+        if LLC_HP_LABEL in self.df:
+            columns += extra_columns
 
-             'memory_bandwidth/local/hp/bytes',
-             'memory_bandwidth/local/be/bytes',
-        ]]
+        df = self.df[columns]
```
Comment (iwankgb) [inline jupyter/swan.py]: @ppalucki - provide docs (link to cheat sheet preferably) and/or rename the function (eg. to `filter_dataframe` or `shrink_dataframe`) please.

File: jupyter/swan.py [hunk 3]
```diff
@@ -770,20 +820,52 @@ def fmtbytes(b):
...
         )
 
-    def pivot_ui(self):
-        return _pivot_ui(self.simple_df())
+    def pivot_ui(self, **options):
+        return _pivot_ui(self.simple_df(), **options)
+
+    def latency(self, normalized=True, aggressor=None, qps=None):
+
+        df = self.df
```
Comment (iwankgb) [inline jupyter/swan.py]: @ppalucki - what is a purpose of this assignment?

File: jupyter/swan.py [hunk 4]
```diff
@@ -579,6 +588,14 @@ def caffe_batches(self):
                 self._get_caption('caffe image batches')
             )
 
+# --------------------------------------------------------------
+# "optimal core allocation" experiment
+# --------------------------------------------------------------
+
+
+NUMBER_OF_CORES_LABEL = 'number_of_cores'  # HP cores.
+SNAP_USE_COMPUTER_SATURATION_LABEL = '/intel/use/compute/saturation'
```
Comment (iwankgb) [inline jupyter/swan.py]: s/COMPUTER/COMPUTE

**Comment by ppalucki** [comment]
run integration tests

File: jupyter/swan.py (hunk 2 above)
Comment (ppalucki) [inline jupyter/swan.py]: added an suitable comment to function to explains purpose of this

**Comment by ppalucki** [comment]
@iwankgb please re-review

**Pull request merged**